docs/organizational-structure.rst

```diff
@@ -45,7 +45,7 @@ Role Contributors
 -----------------
 
 In context of Ansible roles, each role not directly created by DebOps
-Developers needs a Contributor. The role Contributor signs the role using
+Developers is a Contributor. The role Contributor signs the role using
```

Comment:
I introduced Role Contributors when I realized that either Role Author could be someone completely unrelated, or someone who didn't have a GPG key and didn't want to deal with that - since after role is added to the project, Role Maintainer takes over anyway. Any thoughts with that in mind?